torchci/lib/bot/cliParser.ts

```diff
@@ -187,7 +187,7 @@ parser.add_argument("-h", "--help", {
   action: "store_true",
 });
 
-const botCommandPattern = new RegExp(/^ *@pytorch(merge|)bot.*$/m);
```

Comment:
Please post test use case should we use ``\s`` to match the space instead ?

Reply:
> should we use \s to match the space instead

That would match tabs and line breaks also, we want just literal whitespace.

Reply:
Because it mimics command line, you don't use line breaks in command line to separate argument.